src/matchers/app.rs

```diff
@@ -100,3 +105,18 @@ pub fn is_der(buf: &[u8]) -> bool {
 
     buf.len() > 2 && buf[0] == 0x30 && buf[1] == 0x82
 }
+
+/// Returns whether a buffer is a Common Object File Format for i386 architecture.
+pub fn is_coff_i386(buf: &[u8]) -> bool {
+    buf.len() > 2 && buf[0] == 0x4C && buf[1] == 0x01
+}
+
+/// Returns whether a buffer is a Common Object File Format for x64 architecture.
+pub fn is_coff_x64(buf: &[u8]) -> bool {
+    buf.len() > 2 && buf[0] == 0x64 && buf[1] == 0x86
+}
+
+/// Returns whether a buffer is a Common Object File Format for Itanium architecture.
+pub fn is_coff_ia64(buf: &[u8]) -> bool {
+    buf.len() > 2 && buf[0] == 0x00 && buf[1] == 0x02
+}
```

Comment:
I think we should also have a generic `is_coff` function, and register that as a matcher instead of all of the specific ones